ztp/ran-crd/site-config-crd.yaml

```diff
@@ -76,6 +76,22 @@ spec:
                   type: object
                   additionalProperties:
                     type: string
+                crAnnotations:
+                  description: |
+                    An optional CR annotations parameter enables users to patch
+                    the default generated CRs with defined annotations. The annotations
+                    defined in clusters or nodes take precedence over the annotations
+                    at this level.
+                  type: object
+                  properties:
+                    add:
```

Comment:
Same as above why do we need to mentioned Add ? Is it possible to have delete/remove as well ?